### ISSUE TITLE
Why ChunkExtractor is required?

### ISSUE THREAD

**Issue by vadim-ch** [gregberge/loadable-components#992]
Hi there!
createLoadable function includes a condition that checks `props.__chunkExtractor`
https://github.com/gregberge/loadable-components/blob/main/packages/component/src/createLoadable.js#L149
What if i don't want to use ChunkExtractor at all? To be more correctly, i can't add ChunkExtractor, because i use ReactJs.NET on my server.

**Comment by theKashey** [comment]
Without chunk extractor active Loadable will not be able to properly hydrate App as it will not know which chunks are required to be loaded first.

I am not sure how exactly ReactJs.NET works and why you cannot use ChunkExtractor and it's more about webpack rather than platform, but if you still can use Babel try a sibling solution - https://github.com/theKashey/react-imported-component, which does not use anything except Babel

**Comment by vadim-ch** [comment]
As far as I understand, ChunkExtractor is executed during request processing in node.js. It receives loadableStats.json as chunks information. And then a ChunkExtractor instance is passed to the client components using React Context.
So. In this case, the source of truth is loadableStats.json and i can make required scripts by myself based on loadableStats.json. It seems to me that ChunkExtractor is only a helper to create script tags. 
Am i wrong?

**Comment by theKashey** [comment]
Correct, it's only a bearer of mapping between import and chunk. This information is generated by a webpack plugin, but can be generated in any other way.
